scylla/src/statement/mod.rs

```diff
@@ -20,6 +21,8 @@ pub struct StatementConfig {
     pub request_timeout: Option<Duration>,
 
     pub history_listener: Option<Arc<dyn HistoryListener>>,
+
+    pub execution_profile_handle: Option<ExecutionProfileHandle>,
```

Comment:
OK, I see that you add it here. I think the commits should be in a different order.

Reply:
I keep the original order but I've corrected the commit description (in commit that remove policies).